### ISSUE TITLE
recutils compiled without UUID support?

### ISSUE THREAD

**Issue by Thomq** [Homebrew/homebrew#39148]
Installing recutils with homebrew doesn't provide UUID support.  The mailing list for recutils suggests that recutils may have been compiled with UUID support disabled.  Anyone have a clue how to change that so UUID support is enabled?

http://lists.gnu.org/archive/html/help-recutils/2015-04/msg00001.html

**Comment by mistydemeo** [comment]
It looks like recutils needs ossp-uuid to build UUID support, and the formula doesn't specify that dependency.

**Comment by Thomq** [comment]
Ah, okay. Well, just as long as there wasn't a reason for not including the dependency, like some sort of failure to build or incompatibility.

But wow. I have no idea what I'm doing with this github thing anymore. I tried the fork bit. Dead end. I tried deleting the previous fork I had from some year or two ago, then forking again… dead end there, too. Can't simply fork the formula by itself…not obvious to me at least.

I recall github seeming obscure to me before, but wow…I really don't have a clue anymore. I guess github makes sense for those in the know.

I'll see what I can I do manually editing the recutils formula file on my computer to add the dependency. If I can find the file. And if the brew command doesn't protest. Or maybe I'll just skip the uuid for now. Using uuid merely sounded like a neat idea. Supposedly I can have an integer auto-increment instead. I'll probably try that.
